trie/triedb/pathdb/nodebufferlist.go

```diff
@@ -73,6 +74,129 @@ type nodebufferlist struct {
 	forceKeepCh     chan struct{}  // Trigger force keep event loop.
 	waitForceKeepCh chan struct{}  // Wait force keep event loop.
 	keepFunc        NotifyKeepFunc // Used to keep op-proposer output proof.
+
+	freezer  *rawdb.ResettableFreezer // Freezer for storing trie histories, nil possible in tests
```

Comment:
It's better not to add this in nodebufferlist struct.

Reply:
Deleted